chaincode/main.go

```diff
@@ -170,6 +170,12 @@ func (t *SubstraChaincode) Invoke(stub shim.ChaincodeStubInterface) peer.Respons
 	if err != nil {
 		return formatErrorResponse(err)
 	}
+	// Send event if there is any. It's done in one batch since we can only send
+	// one event per call
+	err = db.SendTuplesEvent()
+	if err != nil {
+		return formatErrorResponse(fmt.Errorf("could not send event for unknown reason"))
```

Comment:
Is there a reason not to add the error returned by `SendTuplesEvent` as part of this error, instead of saying the error is unknown?

Reply:
confirmed with @thibaultrobert this needs fixing

Reply:
fixed: https://github.com/SubstraFoundation/substra-chaincode/pull/41/commits/53ccf231c878e305f5aef1e17ca09dc5671690d8